src/main/java/com/authlete/common/api/AuthleteApi.java

```diff
@@ -621,7 +620,6 @@ TokenListResponse getTokenList(String clientIdentifier, String subject,
      * </p>
      *
      * <pre>
-     * int totalCount = api.{@link #getTokenList(String, String, int, int, TokenStatus)
```

Comment:
Oh, no. I meant that you had to either

1. change `#getTokenList(String, String, int, int, TokenStatus)` to `#getTokenList(String, String, int, int)` or
2. change `getTokenList(clientIdentifier, subject, 0, 0)` to `getTokenList(clientIdentifier, subject, 0, 0, TokenStatus.VALID)` (`TokenStatus.VALID` can be another `TokenStatus` enum entry).

Didn't you notice that removing this line would break the JavaDoc? Be careful.